pkgs/native_toolchain_c/test/clinker/test_data/linker/symbols.lds

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Sorry for being ambiguous. I mean a `test_data/` directory in the package directory, not nested in the test directory. That's the structure we're using so far. See the reasoning here:

* https://github.com/dart-lang/native/pull/938

I believe `test/data/` is the more common place to do things, but since we have dart projects as test data that messes up things.  If you're mostly confident that we'll never have Dart projects as test data, we can also use `native_toolchain_c/test/data/`, but maybe it's better to just be consistent and go with `native_toolchain_c/test_data/`

Reply:
Ah ok. That is a larger change, I would rather do that in a separate PR.

Reply:
Well, if you want to do it in a separate PR, please un-move everything in this PR 😄